tox.ini

```diff
@@ -53,5 +53,5 @@ passenv =
     MYPY_FORCE_COLOR
     MYPY_FORCE_TERMINAL_WIDTH
 commands =
-    python -m mypy --config-file mypy_self_check.ini -p mypy -p mypyc
-    python -m mypy --config-file mypy_self_check.ini misc --exclude misc/fix_annotate.py --exclude misc/async_matrix.py --exclude misc/sync-typeshed.py
+    python runtests.py self
```

Comment:
The current command also runs the self-check on the `mypyc/` directory; `python runtests.py self` doesn't